android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -81,38 +81,46 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
         return "RNOneginiSdk"
     }
 
+    private fun Promise.rejectWithNullError(paramName: String, paramType: String){
+        this.reject("error_code", "Expected param '$paramName' to be '$paramType' but was NULL")
```

Comment:
I missed that before but, there should be a proper error code returned 😅 Also I'm thinking about extracting the error message to some constants, similar to how it's done in the SDK.